lib/scsi-lowlevel.c

```diff
@@ -1487,11 +1487,6 @@ scsi_inquiry_unmarshall_device_identification(struct scsi_task *task)
 	return inq;
```

Comment:
Please use the imperative mood for patch descriptions. That means changing `cleaned a dead codes` into e.g. `Remove dead code`. Otherwise this patch looks good to me.